range-v3/include/range/v3/view/cycle.hpp

```diff
@@ -78,37 +78,43 @@ namespace ranges
                 using cycled_view_t = constify_if<cycled_view>;
                 using difference_type_ = range_difference_t<Rng>;
                 using iterator = range_iterator_t<constify_if<Rng>>;
+                using sentinel = range_sentinel_t<constify_if<Rng>>;
 
-                cycled_view_t *rng_;
+                // cycled_view_t *rng_;
```

Comment:
We have to disable something from range-v3 to get this to work? if so, we need to prepare a small example and a justification of the problem so we can report it or at least keep track of it. 
If ranges get accepted on C++20 we need to try to make it as compatible as possible with parallel stl

Reply:
Yes, I had to remove the pointer field, since it can't be captured as a kernel argument. Unfortunately, I couldn't figure out how to work around everything, only the bits I needed. Michael added a [diff](https://github.com/codeplaysoftware/parallelts-range-exploration/blob/master/range-v3/sycl_range_v3.diff) of the changes he made and we've both made some others. Where should we keep track of these?